lib/rule_blocklist.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+const a = "a"
+  , Config = require("eslint/lib/config")
+  , merge = require("eslint/lib/config/config-ops").merge
+  ;
+
+const blocklistedRules = {
+  "no-unresolved": "Code Climate doesn't fetch your project's dependencies so this rule produces high amount of false positives."
+};
+
+function filterRules(rules) {
+  let report = [];
+
+  for (const name in blocklistedRules) {
+    if (Reflect.has(rules, name)) {
```

Comment:
It's a [standard built-in object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect).